### PR TITLE
fix(vapt): deny x-frame-options

### PR DESCRIPTION
**Pull request by tyng94**
Fix clickjacking issue for maps by enabling x-frame-options header in configuration

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -29,19 +29,27 @@ def setup_app_conf(app: Flask, prefix: str) -> None:
...
+    #                 Defaults to the 'static' folder in the root path of the application.
+    FLASK_APP_ENV_PREFIX = "FLASK_APP_"
+    app = Flask(__name__)
 
-setup_app_conf(app, FLASK_APP_ENV_PREFIX)
-Compress(app)
+    @app.after_request
+    def apply_caching(response):
+        response.headers["X-Frame-Options"] = "DENY"
+        return response
```
Comment (tyng94) [inline app.py:45]: fyi, this is the only meaningful change

**Comment by tyng94** [comment]
an alternative is to completely take down this server

**Review by Tingweiftw — APPROVED** [review]
LGTM

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -30,7 +30,6 @@ jobs:
         if [[ "${EVENT}" == "push" || "${EVENT}" == "release" ]]; then IMAGE_TAG="${REF_TAG}"
         else IMAGE_TAG="test-${REF_TAG}"; fi
 
-        echo "::set-env name=IMAGE_TAG::${IMAGE_TAG}"
```
Comment (Tingweiftw) [inline .github/workflows/ci.yml:33]: why is this removed?

**Comment by tyng94** [comment]
won't do, will instead disable the fover server
